### DISCUSSION TITLE
IPAM :( Application Error

### DISCUSSION THREAD

**Discussion by tjh66** [Azure/ipam#272] [General]
Hi I have installed the IPAM solution but when browsing to the URL I get the following error below
**:( Application Error**
I did have an uft-8 error initially at the end of the deployment but over come that by using the setting in the O/S  **Beta: Use Unicode UTF-8 for worldwide language support**  everything has installed ok am I missing some major steps after solution deployment? tia

**Comment by DCMattyG** [comment]
Hi @tjh66 , happy to assist!

Can you please provide some additional context such as how exactly you deployed the Azure IPAM solution?

What was the exact command line that you used for the deployment?

Have you verified all of the [prerequisites](https://azure.github.io/ipam/#/deployment/README?id=prerequisites) are met from the Deployment guide?

There are usually some clues as to what has failed in the deployment/error logs which are located in the "logs" folder of the directory after the deployment has completed. I'd be happy to review the log files further if you can share them here, or you're also welcome to email them to me directly if you feel more comfortable with that.

Feel free to drop me a line at Matthew.Garrett@microsoft.com if you'd prefer to discuss via email, otherwise please share some additional details here and we'll help figure out what is wrong as quickly as we can.

Thanks!

**Comment by DCMattyG** [comment]
Hi @tjh66, just following-up with you on this item. I believe you were running into this due to a few bugs that have now been addressed in PR #279 and are incorporated into the latest release (v3.2.0) of Azure IPAM.

I believe you also sent me an email (which I responded to) on this topic.

Hopefully you'll be able to resolve everything now with the fixes in place, but if not please let me know and I'm happy to work with you to make sure everything is successfully up and running and I can answer any questions you might have about using the product.

Have a great one and I hope these fixes have addressed all of your issues!

**Reply by tjh66** [reply]
Hi @DCMattyG thanks for the response I am OOO until Tuesday but will give it a try then, thanks for the support and I will be in touch good or bad :)

**Reply by tjh66** [reply]
@DCMattyG Thanks for you help with this redeployed and everything is working as expected.

**Reply by DCMattyG** [reply]
So great to hear, thanks for confirming!
